homeassistant/components/isy994.py

```diff
@@ -22,6 +22,7 @@
 # homeassistant constants
 DOMAIN = "isy994"
 DEPENDENCIES = []
+REQUIREMENTS = ['PyISY>=1.0.2']
```

Comment:
Please update this to PyISY?=1.0.5. After this, my update to the PyISY module should merge cleanly.